kingpin/actors/rightscale/test/test_server_array.py

```diff
@@ -546,24 +546,20 @@ def test_launch_instances(self):
         launch = mock_tornado()
         self.client_mock.launch_server_array.side_effect = launch
 
-        self.client_mock.get_server_array_current_instances = mock_tornado([])
-
-        two_mock_calls = [mock.call(array_mock), mock.call(array_mock)]
-        four_mock_calls = [
-            mock.call(array_mock), mock.call(array_mock),
-            mock.call(array_mock), mock.call(array_mock)]
-
         # Regular function call
+        self.client_mock.get_server_array_current_instances = mock_tornado([])
         self.client_mock.launch_server_array.reset_mock()
         yield self.actor._launch_instances(array_mock)
-        self.client_mock.launch_server_array.assert_has_calls(four_mock_calls)
+        self.client_mock.launch_server_array.assert_has_calls(
```

Comment:
Here I disagree. The code in the Actor module has no loop in it at all -- its not possible that it would be called multiple times. Instead, I'm just making sure that the right `count` was passed in.